.github/workflows/deploy_to_railway.yml

```
@@ -29,9 +29,16 @@ on:
+    environment: 'Development'
+    env:
+      RAILWAY_TOKEN: ${{ secrets.RAILWAY_GITHUB_ACTIONS }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-      - name: Do nothing
-        run: echo "Do nothing here"
+      - name: Install Railway CLI
+        run: npm i -g @railway/cli
+      - name: Apply new image to Railway
+        run:  railway up --service=apostrophe-cms
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)